main.go

```diff
@@ -59,12 +62,15 @@ func main() {
 		KeyFilePath:         keyFilePath,
 	}
 
+	eventDispatcher := mdevent.NewEventDispatcher(EventBus.New())
```

Comment:
Well, I guess we have to add `EventBus` interface now. we are managing dependencies directly inside `main.go`. This may confuse people when they want to switch to remote `pubSub` but can't find it inside `dep/wire.go`.

Reply:
I agree with you. Maybe we will add `EventBus` adapter when we have an appropriate case? I mean, we should design the `EventBus` interface carefully.